jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/ImageProvider.java

```diff
@@ -300,9 +300,10 @@ public void invalidateAll() {
 
 	private File findFileInPaths(String fileName) {
 		for (File path : this.lookupPaths) {
-			File searched = new File(path, fileName);
-			if (searched.isFile() && searched.canRead()) {
-				return searched;
+			for (File f : path.listFiles()) {
```

Comment:
Please don't use single letter variable names for this. Calling it currentFile is more obvious (I know, here isn't much code around, but we want to keep a clean code in the project ;) )